smart-metastore/src/main/java/org/smartdata/metastore/dao/AccessEventAggregator.java

```diff
@@ -86,57 +93,57 @@ private void createTable() {
       LOG.error("Create table error: " + table, e);
```

Comment:
Error messages could be more informative regarding the point of failure. Right now it would be difficult to debug any issues based on these logs on your own. It would be nice to include more contextual information about where the error occurred in the logged message.